matrix/all-in-one.yaml

```diff
@@ -0,0 +1,534 @@
+component: Splunk Distribution of OpenTelemetry .NET
+version: 1.0.2
+dependencies:
```

Comment:
name (human readable) is missing

Reply:
2153f202b9a75ad5393ce0bdceaeac32559f4f31